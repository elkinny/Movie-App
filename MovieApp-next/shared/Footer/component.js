import React from 'react';

import styles from './style.scss';

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <span className={styles['footer__text']}>Copyright elkin (c) 2019</span>
    </footer>
  );
};

export default Footer;
