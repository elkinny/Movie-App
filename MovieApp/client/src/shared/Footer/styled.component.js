import styled from 'styled-components';

export const StyledFooter = styled.footer`
  width: 100%;
  padding: 20px;
  display: flex;
  align-content: center;
  justify-content: center;
`;

export const StyledText = styled.span`
  color: #7488ab;
`;
