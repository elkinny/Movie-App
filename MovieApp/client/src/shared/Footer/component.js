import React from 'react';

import { StyledFooter, StyledText } from './styled.component.js';

const Footer = () => {
  return (
    <StyledFooter>
      <StyledText>Copyright elkin (c) 2019</StyledText>
    </StyledFooter>
  );
};

export default Footer;
