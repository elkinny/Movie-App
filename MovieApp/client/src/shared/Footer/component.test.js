import Footer from './component';
import React from 'react';
import { mount } from 'enzyme';

describe('Footer: ', () => {
  it('Footer to match snapshot', () => {
    const component = mount(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
