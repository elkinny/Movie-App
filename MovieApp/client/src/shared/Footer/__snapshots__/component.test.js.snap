// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer:  Footer to match snapshot 1`] = `
<Footer>
  <footer
    className="footer"
  >
    <span
      className="footer__text"
    >
      Copyright elkin (c) 2019
    </span>
  </footer>
</Footer>
`;
